docs/pages/textarea.js

```diff
@@ -317,6 +576,21 @@ function Example(props) {
       `}
         />
       </MainSection>
+
+      <MainSection name="Related">
+        <MainSection.Subsection
+          description={`
+      **[TextField](/textfield)**
+      TextField is ideal for short-form, single answer text input.
+
+      **[Tag](/tag)**
+      Tag can be used in conjunction with TextArea to display separate elements of content.
+
+      **[ComboBox](/combobox)**
+      ComboBox + Tag is the recommended alternative to TextArea + Tag when selecting from a finite list list of items.
```

Comment:
Consider adding a link to "TextArea + Tag" to the example above, as that usage may not be intuitive

Reply:
Done